packages/data-context/src/sources/HtmlDataSource.ts

```diff
@@ -60,16 +74,23 @@ export class HtmlDataSource {
   }
 
   async makeServeConfig () {
+    const cfg = this.ctx._apis.projectApi.getConfig() ?? {} as any
```

Comment:
Yeah we need to make sure we have a single place to access this… also instead of passing the whole config, if we can just psss what we need, that might be good.

Reply:
Hopefully we can change this to something like `serverCypressInCypressConfig` which just has the minimal data like ports/namespace. Likely we will revisit this as Cypress in Cypress is still a work in progress.